src/lib/basic_graph/basic-graph.js

```diff
@@ -13,6 +13,7 @@ export default class BasicGraph {
     return new BasicGraph(itemSubset, edgeSubset)
   }
 
+  // TODO(Question for Ozzie): Why aren't these passing oneLevel down? Seems like this shouldn't work...
```

Comment:
you're right, seems like a mistake.  My strong guess is that we don't yet have a case where onelevel is false